### PR TITLE
Pull up to arbor master and add CI

### PR DESCRIPTION
**Pull request by thorstenhater**

### CODE REVIEW

File: install-local.sh [hunk 1]
```diff
@@ -194,8 +194,8 @@ fi
 
 # Build simulator targets.
 
-export CC="$ns_cc"
-export CXX="$ns_cxx"
+# export CC="$ns_cc"
+# export CXX="$ns_cxx"
```
Comment (noraabiakar) [inline install-local.sh]: Are these meant to stay commented?

**Review by noraabiakar — COMMENTED** [review]
Looks good! Thanks. Just one small question

File: install-local.sh (hunk 1 above)
Comment (thorstenhater) [reply install-local.sh]: Probably not, but I had specific issues with it on Juwels. I'll revert it and fix as it comes back up.

Comment (thorstenhater) [reply install-local.sh]: Done @noraabiakar

**Review by brenthuisman — CHANGES_REQUESTED** [review]
To avoid issues with shifting ~~sands~~submodules, cloning the requested tag instead of master will help (or was this solved another way?).

In `build_arbor.sh` to `git clone --depth 1 --branch <tag_name> <repo_url>`.

**Pull request merged**